src/cluster_legacy.c

```diff
@@ -1755,7 +1755,7 @@ void clusterHandleConfigEpochCollision(clusterNode *sender) {
     server.cluster->currentEpoch++;
     myself->configEpoch = server.cluster->currentEpoch;
     clusterSaveConfigOrDie(1);
-    serverLog(LL_VERBOSE,
+    serverLog(LL_WARNING,
               "WARNING: configEpoch collision with node %.40s (%s)."
               " configEpoch set to %llu",
```

Comment:
Will this warning scare users?

Normally an epoch collision is resolved automatically. Maybe use NOTICE here too?